lessons/module-2/context-and-this-keyword.md

```diff
@@ -25,32 +25,32 @@ As we have discussed earlier, the keyword *this* in JavaScript can be confusing.
 Consider the following `Unicorn` class:
 
 ```javascript
-class Unicorn {
-  constructor(name, color) {
-    // What is logged here?
-    console.log(this);  
-
-    this.name = name;
-    this.color = color;
-
-    // What changes about "this"?
-    console.log(this);  
-  }
-
-  says() {
-    console.log(`${this.name} is my favorite ${this.color} pony`, this);
+function createUnicorn(name, color) {
+  // What is logged here?
+  console.log('value of this:', this)
+  
+  
+  return {
+    name: name,
+    color: color,
+    says: function says() {
+    console.log(`${this.name} is my favorite ${this.color} pony, `, 'value of this:', this)
   }
+  // what changes about `this` inside of the object?
+  }  
 }
+
+var newUnicorn = createUnicorn('Baby', 'Black and White')
 ```
 
-* Create a `new` instance of the `Unicorn`.  What rule can we infer about *this* when creating a `new` instance?
+* Create a new `unicorn` object.  What rule can we infer about *this* when creating an object?
 * Now try running it's method, `says`.  What does *this* refer to now?  Has it changed?
 </section>
 
 <section class="answer">
 ### Rule #1  
 
-*this* within function code invoked using the `new` operator refers to the new instance of that object.
+The value of *this* within a standalone function will refer to the global object - in JavaScript, this is the [global `Window` object.](https://developer.mozilla.org/en-US/docs/Web/API/Window)
```

Comment:
I appreciate the addition to the MDN docs for the `Window` object.  They don't need to get super in depth with it, but for those that are curious, they can read more about it there!